ComSemApp/templates/ComSemApp/student/assessment.html

```diff
@@ -182,9 +190,7 @@ <h2 id="timer-text" class="show-expression"></h2>
         'X-CSRFToken': csrf_token
       },
       success:(response) => {
-        cs_notification('success', "Attempt Saved")
-        // inputs the id recieved into the html form
-        $("#form").append("<input type='hidden' value="+response['id']+" name='attempt_ids'></input>");
```

Comment:
not needed because we now have the session id?

Reply:
yes